uint8 id1
uint8 id2
uint8 id3
---
int32 position1
int32 position2
int32 position3